swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        
+        if id is None:
+            unique_id = uuid.uuid4()
+            self.id = unique_id.int
+        else:
+            self.id = id         
+
+        self.condition = condition      
+
+    def get_category(self):
+        return "Item"   
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."    
+    
+    def condition_description(self):
+        descriptions = {
+            0: "Heavily used",
+            1: "Gently used",
+            2: "Fair",
+            3: "Good",
+            4: "Very good",
+            5: "Like new"
+        }
+        return descriptions.get(self.condition)
```

Comment:
👀 There is a test case `test_items_have_condition_as_float` which implies that we should support conditions as floats. Even though there's no specific test for float values and descriptions, from reading the existing tests, this is something we should consider (and potentially even add a test for!).

As written, think about what this code would do if the condition were a float value falling between one of these values. I.e., what would this function return with a condition of `3.5`? If we assume `3.5` is a valid condition (as the test implies), would that result make sense, or would a client of our class find that confusing? How could we update this logic to provide a result more aligned with treating `3.5` as a valid condition?